### PR TITLE
Introduce Select/Checkbox/Website/Hidden form fields

### PR DESCRIPTION
**Pull request by EvanHerman**
Introduce new form fields:
- [x] Select
- [x] Checkbox
- [x] Website (URL)
- [x] Hidden

### CODE REVIEW

File: includes/class-coblocks-form.php [hunk 1]
```diff
@@ -452,6 +426,12 @@ public function render_field_radio( $atts, $content ) { // phpcs:ignore Generic.
 
 		$this->render_field_label( $atts, $label );
 
+		if ( $atts['isInline'] ) {
```
Comment (paranoia1906) [inline includes/class-coblocks-form.php]: I am getting 
`Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 429` 
and 
`Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 448`


Should be able to resolve this by using `if ( isset( $atts['isInline'] ) )`

Comment (jrtashjian) [reply includes/class-coblocks-form.php]: Resolved in 1ebdccc

File: includes/class-coblocks-form.php [hunk 2]
```diff
@@ -465,6 +445,166 @@ public function render_field_radio( $atts, $content ) { // phpcs:ignore Generic.
 
 		}
 
+		if ( $atts['isInline'] ) {
```
[the remark this answers is not in the document]
Comment (jrtashjian) [reply includes/class-coblocks-form.php]: Resolved in 1ebdccc

**Comment by EvanHerman** [comment]
> I am getting
> `Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 429`
> and
> `Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 448`
> 
> Should be able to resolve this by using `if ( isset( $atts['isInline'] ) )`

I wonder if this is because you tested it before I got around to adding the `isInline` attribute? I'll flush out my plugin and re-test this before merging to double check.

**Comment by EvanHerman** [comment]
> I am getting
> `Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 429`
> and
> `Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 448`
> 
> Should be able to resolve this by using `if ( isset( $atts['isInline'] ) )`

@paranoia1906 Nice catch. This was an issue that I also did see in when running our unit tests.

File: includes/class-coblocks-form.php [hunk 3]
```diff
@@ -492,16 +632,27 @@ public function render_field_label( $atts, $field_label ) {
 		$required_label = empty( $required_attr ) ? '' : sprintf( ' <span class="required">%s</span>', $required_text );
```
[the remark this answers is not in the document]
Comment (EvanHerman) [reply includes/class-coblocks-form.php]: Test have been added for all new fields 👌

File: .dev/tests/phpunit/includes/test-coblocks-body-classes.php [hunk 4]
```diff
@@ -66,7 +66,6 @@ public function test_is_active_theme() {
 	public function test_themes() {
 
 		$expected = [
-			'twentytwenty',
```
Comment (richtabor) [inline .dev/tests/phpunit/includes/test-coblocks-body-classes.php]: Should this be removed? I'm unsure when this happened?

**Comment by richtabor** [comment]
@EvanHerman Mind double-checking that last commit didn't mess up anything? Thanks!

**Comment by EvanHerman** [comment]
@richtabor Not sure what happened there in that last merge, but it does look like something went awry. I'll try and see if I can figure it out.

**Comment by EvanHerman** [comment]
@richtabor I reverted back to before the `17814ef` merge. It seems something went wrong there. I don't think we need to merge `master` into here, there are no merge conflicts. We should just be good to merge.

**Pull request merged**